docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md

```diff
@@ -29,22 +29,22 @@ There are two types of keys that are used to encrypt data in YugabyteDB:
 - Universe key: The top-level symmetric key used to encrypt other keys and are common to the universe.
 - Data key: The symmetric key used to encrypt data. There is one data key generated per flushed file.
 
-For details on the features, assumptions, design, data key management, universe keys, key rotations, master failures, and adding a node, see [Encryption At Rest in YugabyteDB](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-encryption-at-rest.md).
+For more information on the features, assumptions, design, data key management, universe keys, key rotations, master failures, and adding a node, see [Encryption At Rest in YugabyteDB](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-encryption-at-rest.md).
 
 ## Enable encryption at rest during universe creation
 
 You can enable encryption at rest during the universe creation as follows:
 
-1. Open the Yugabyte Platform console and navigate to **Universes**.
-2. Click **Create Universe**. The **Universe Configuration** page appears.
+1. Open the Yugabyte Platform UI and navigate to **Universes**.
+2. Click **Create Universe** to open the **Universe Configuration** page.
 3. After you select a provider, the **Instance Configuration** section expands to show more options.
 4. Select the **Enable Encryption at Rest** option. The **Key Management Service Config** option appears.
-5. Select your KMS configuration from the **Key Management Service Config** list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration using AWS KMS](../create-kms-config.md/aws-kms/) and [Create a KMS configuration using Equinix SmartKey](../create-kms-config.md/equinix-smartkey/).
+5. Select your KMS configuration from the **Key Management Service Config** list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration using AWS KMS](../create-kms-config/aws-kms/) and [Create a KMS configuration using Equinix SmartKey](../create-kms-config/equinix-smartkey/).
```

Comment:
Add Hashicorp link? Or replace all three with one generic one pointing to aws?

Reply:
Added link